src/components/PostCard/PostCard.tsx

```diff
@@ -9,25 +9,30 @@ import {SuggestPostInfo} from '../PostInfo/SuggestPostInfo';
 
 import './PostCard.scss';
 
-const b = block('post-card');
+type PostCardSize = 's' | 'm';
+type TitleHeadingLevel = 'h2' | 'h3';
```

Comment:
Can we make an enum for these values ? And we can use it in another files https://github.com/gravity-ui/blog-constructor/pull/103/files#diff-112399abf9eaac816425b4c2015f063a593c518e0ccff35b04814de737bae3beR59

Reply:
done